core/gotemplate/Readme.md

```diff
@@ -64,34 +153,33 @@ gotemplates:
 ```
 
 # Static assets
-You can use Flamingo’s built-in static file handler to serve static assets that you need in your template.
+You can use Flamingo’s built-in static file handler to automatically serve necessary static assets from your asset folder.
 
-Set it up by adding a route with a param called “name” (that will get the name of the asset), such as 
+You can set it up by adding a route and setting the `name` param to the name of your asset folder.
 
-```
-polls/urls.go:
-```
+In the following example, our assets lie in the `asset` folder:
 
 ```go
+// /polls/urls.go
 func (u *urls) Routes(registry *web.RouterRegistry) {
     // ...
     registry.MustRoute("/asset/*name", `flamingo.static.file(name, dir?="asset")`)
 }
 ```
 
-Or via the routes.yml feature:
-
-Edit your file `config/routes.yml`
+Or via your routes.yml configuration file:
 
 ```yaml
+# /config/routes.yml
 - controller: flamingo.static.file(name, dir?="asset")
   path: /asset/*name
 ```
 
-Then, in your template, create the url by doing:
+Then, set up a reference to the the url by adding the following script to your template:
```

Comment:
Perhaps leave that whole polls stuff out and just mention the template function `url`